db/migrate/20190708192323_fix_unserializable_notification_options.rb

```diff
@@ -0,0 +1,44 @@
+class FixUnserializableNotificationOptions < ActiveRecord::Migration[5.1]
+  class Notification < ActiveRecord::Base
+    belongs_to :subject, :polymorphic => true
```

Comment:
This is really dangerous because it can load arbitrary classes

Reply:
Yeah, did you see my comment above?  If we abandon trying to recover the data and blow away options, we don't need this.  I was trying to capture anything I could from the existing notification.

Reply:
I read your comment, but it sounded like enumerating all classes was still a possibliity.

Reply:
If we decide it's worth it to try to get this information corrected, I could remove the `belongs_to :subject` association and instead determine the `migration_stub` model from the `subject_type`, then find it using the stub using the `subject_id`... that should work as long as I enumerate all the base models as stubs.  Is this worth it?